src/physics/cam/physpkg.F90

```diff
@@ -157,6 +159,8 @@ subroutine phys_register
 
     ! Get physics options
     call phys_getopts(shallow_scheme_out       = shallow_scheme, &
+                      deep_scheme_out          = deep_scheme, &
```

Comment:
As discussed, `deep_scheme` is no longer needed, so it can be removed -- either do it directly, or add an issue

Reply:
OK, this shouyld be done in https://github.com/m2lines/CAM-ML/pull/2/commits/dbea84f70311b44441c0a0d1ef07977ac97241c9